garage/config_personal_template.py

```diff
@@ -45,3 +45,9 @@
 AWS_ACCESS_KEY = os.environ.get("AWS_ACCESS_KEY", "<insert aws key>")
 
 AWS_ACCESS_SECRET = os.environ.get("AWS_ACCESS_SECRET", "<insert aws secret>")
+
+GCS_CREDENTIAL_PATH = "<insert file path to key>"
```

Comment:
load the key directly into memory from the environment (see AWS keys above) rather than using a file key--this makes it easier to use in Docker or on cloud hosts.